src/Perpetuum/Zones/SessionlessPlayerTimeout.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Perpetuum.Log;
+using Perpetuum.Players;
+using Perpetuum.Timers;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Perpetuum.Zones
+{
+    public class SessionlessPlayerTimeout
+    {
+        private static readonly TimeSpan MAX_ORPHAN_TIME = TimeSpan.FromMinutes(6);
+        private static readonly TimeSpan UPDATE_RATE = TimeSpan.FromMinutes(2);
```

Comment:
Zone.update happens every 30s, but thought it would be good to decouple that update-rate from this one as the need for this to update frequently is not high.